### ISSUE TITLE
Expand on the file_format tag

### ISSUE THREAD

**Issue by ZerithZA** [vn-ki/anime-downloader#428]
The renaming functionally in file_format is extremely lacking. Is there a way to disable it? I'd rather have a name of "Ajin.S01E01.1080p.Blu-Ray.10-Bit.Dual-Audio.DTS-HD.x265-iAHD.mp4" than having it changed to ajin_01.mp4. It drops so much of the useful information from the file names.

**Comment by Blatzar** [comment]
Shouldn't be very hard to add, just a matter of if this is deemed as unnecessary crust by @vn-ki

**Comment by Blatzar** [comment]
Will give code example to you tomorrow when I'm at pc

**Comment by ZerithZA** [comment]
Yeah, I've had a quick look and guessing it has to do with either:

`cmd_dict = {
        '{aria2}': 'aria2c {stream_url} -x 12 -s 12 -j 12 -k 10M -o '
                   '{file_format}.mp4 --continue=true --dir={download_dir}'
                   ' --stream-piece-selector=inorder --min-split-size=5M --referer={referer} --check-certificate=false --user-agent={useragent}',
        '{idm}'  : 'idman.exe /n /d {stream_url} /p {download_dir} /f {file_format}.mp4'
    }`

or 

`"cmd_opts": [
                "{stream_url}", "-x", "12", "-s", "12",
                "-j", "12", "-k", "10M", "-o", "{file_format}",
                "--continue", "true", "--dir", "{download_dir}",
                "--stream-piece-selector", "inorder", "--min-split-size",
                "5M", "--referer", "{referer}"
            ]`
Trying to research what should be past in its stead.

**Comment by Blatzar** [comment]
Yeah removing file_format and the flags with it should solve it.

**Comment by vn-ki** [comment]
I think file_format can have an entry which exposes the original name, so that it can be configured in config.

**Comment by Blatzar** [comment]
Could probably get the real video name from https://github.com/vn-ki/anime-downloader/blob/624e15c2cb7fc565f0fe37cc5b0d37b8bc359b7e/anime_downloader/util.py#L305
